### PR TITLE
Upgrade to Polymer 2

### LINKED ISSUE #4: Make responsive

### PR DESCRIPTION
**Pull request by TimvdLippe**
Upgrade to Polymer 2. I was using this playground for some tests, so figured I could just upgrade it anyways :smile: 

image

The only things that I couldn't get to work is the sub-menu's and the animations in `paper-dropdown-menu`. @e111077 do you have an idea there?

Fixes #4

### CODE REVIEW

File: elements.html [hunk 1]
```diff
@@ -1,9 +1,15 @@
+<script src="bower_components/web-animations-js/web-animations.min.js"></script>
```
Comment (ebidel) [inline elements.html]: can you put this in the main page?

File: index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <head>
   <title>Polymer material design elements demos</title>
   <script src="bower_components/webcomponentsjs/webcomponents-lite.min.js" async></script>
-  <link rel="import" href="elements.build.html">
+  <link rel="import" href="elements.html">
```
Comment (ebidel) [inline index.html]: any reason not to stick with the built version?

Comment (TimvdLippe) [inline index.html]: Nope, just forgot to build it :smile:

**Comment by TimvdLippe** [comment]
Oh wait, I built locally with the wrong Polymer version. 1 sec!

**Comment by ebidel** [comment]
Do you mind updating the screenshot in the readme?

**Comment by TimvdLippe** [comment]
Done :+1:

**Comment by ebidel** [comment]
So good!

**Comment by e111077** [comment]
discussion continued offline: seems that the minified web-animations library was not playing nice. Used the next-lite.js polyfill instead

**Pull request merged**